apps/language_server/lib/language_server/providers/code_lens/test.ex

```diff
@@ -0,0 +1,218 @@
+defmodule DescribeBlock do
```

Comment:
Can this module be changed to be namespaced under ElixirLS?
e.g.
```suggestion
defmodule ElixirLS.LanguageServer.Providers.CodeLens.Test.DescribeBlock do
```
This is to prevent possible conflicts with modules defined in a user's project. We may also want to extract this module into a new file.